src/Layout/GuestLayout/Navbar/styles.ts

```diff
@@ -0,0 +1,50 @@
+import { Theme } from '@mui/material/styles/createTheme';
+import { SxProps } from '@mui/system';
+import { CSSProperties } from 'react';
+
+
+export const navbarStyle: CSSProperties = Object.freeze({
```

Comment:
it is generic function, use Object.freeze<CSSProperties> to get advantage of type suggestion

Reply:
done